pkgs/tools/filesystems/catcli/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonApplication
+, docopt, anytree 
+}:
+
+buildPythonApplication rec {
+
+  pname = "catcli";
+  version = "v0.5.13";
+  
```

you've got some trailing whitespace here
```suggestion
, docopt, anytree
}:

buildPythonApplication rec {

  pname = "catcli";
  version = "v0.5.13";

```